test/test_helpers.py

```diff
@@ -136,6 +136,29 @@ def test_outgoing(self):
                     'http://sbolstandard.org/testfiles/_4_FPs'}
         self.assertEqual(outgoing_links(doc), expected)
 
+    def test_find_feature(self):
+        test_dir = os.path.dirname(os.path.realpath(__file__))
+        test_file = os.path.join(test_dir, 'test_files', 'BBa_J23101.nt')
+        doc = sbol3.Document()
+        doc.read(test_file)
+        # create the main component which contain the features then add it to sbol3 Document
+        i13504 = sbol3.Component('i13504', sbol3.SBO_DNA)
+        i13504.name = 'iGEM 2016 interlab reporter'
+        i13504.description = 'GFP expression cassette used for 2016 iGEM interlab study'
+        i13504.roles.append(sbol3.SO_NS + '0000804')
+        doc.add(i13504)
+        # creating a feature and add it to the sbol3 Document
+        b0034 = sbol3.Component('B0034', sbol3.SBO_DNA)
+        b0034.name = 'RBS (Elowitz 1999)'
+        b0034.roles = [sbol3.SO_NS + '0000139']
+        doc.add(b0034)
+        i13504.features.append(sbol3.SubComponent(b0034))
+
+        result=find_feature(i13504,b0034)
+        self.assertEqual(find_feature(i13504,b0034),result)
```

Comment:
This test does not actually check if the function is operating correctly. How would you know if result is actually the right result?